src/components/TokenTranslationUpdate/TokenTranslationUpdate.js

```diff
@@ -0,0 +1,275 @@
+      const allOccurance = props.tokenDetail.occurrences;
+      const data = [
+        { token: token, occurrences:allOccurance , translation: tokenTranslation },
+      ];
+      tokenApiCall(data);
+      handleDialogClose();
+    }else if(replaceType === 'remainingTokens'){
+      const remOccurance = (props.tokenDetail.occurrences).slice(tokenSelected-1);
+      const data = [
+        { token: token, occurrences: remOccurance, translation: tokenTranslation },
+      ];
+      tokenApiCall(data);
+      handleDialogClose();
+    }else{
+      console.log("no types")
+    }
+        
+  };
+
+  return (
+    <Grid container direction='row'>
+      <SnackBar responseStatus={responseStatus} handleClose={handleClose} />
+      <Grid item md={4}>
+        <TextField
+          id='outlined-password-input'
+          autoComplete='current-password'
+          variant='outlined'
+          size='small'
+          value={tokenTranslation}
+          onChange={(e) => setTokenTranslation(e.target.value)}
+        />
+      </Grid>
+
+      <Grid className={classes.icon}>
+        <Tooltip title='Back'>
+          <IconButton aria-label='close' onClick={previous}>
+            <NavigateBeforeIcon className='circle' fontSize='small' />
+          </IconButton>
+        </Tooltip>
+      </Grid>
+
+      <Grid className={classes.tokeninfo}>
+        <span>
+          {' '}
+          {tokenSelected} of {tokenTranslationCount}
+        </span>
+      </Grid>
+
+      <Grid className={classes.icon}>
+        <Tooltip title='Next'>
+          <IconButton aria-label='close' onClick={next}>
+            <NavigateNextRoundedIcon fontSize='small' />
+          </IconButton>
+        </Tooltip>
+      </Grid>
+
+      <Grid className={classes.icon}>
+        <Tooltip title='Replace'>
+          <IconButton
+            aria-label='close'
+            onClick={() => {
+              tokenReplace('oneToken');
+            }}
+          >
+            <DoneIcon fontSize='small' />
+          </IconButton>
+        </Tooltip>
+      </Grid>
+
+      <Grid className={classes.icon}>
+        <Tooltip title='Replace All'>
+          <IconButton aria-label='close' onClick={handleClickOpen}>
+            <DoneAllIcon fontSize='small' />
+          </IconButton>
+        </Tooltip>
+
+        <Dialog
+          onClose={handleDialogClose}
+          aria-labelledby='customized-dialog-title'
+          maxWidth='sm'
+          open={open}
+        >
+          <Grid container direction='row' className={classes.DialogContainer}>
+            <Grid className={classes.closeButtonGrid} item md={12}>
+              <IconButton
+                fontSize='small'
+                aria-label='close'
+                className={classes.closeButton}
+                  onClick={handleDialogClose}
+              >
+                <CloseIcon />
+              </IconButton>
+            </Grid>
+            <Grid item md={7} className={classes.tokenReplace}>
```

Comment:
A message can be added to the dialog to expain the 2 choices below in a sentence.